app/serializers/activitypub/note_serializer.rb

```diff
@@ -164,4 +189,34 @@ def name
 
   class CustomEmojiSerializer < ActivityPub::EmojiSerializer
   end
+
+  class OptionSerializer < ActiveModel::Serializer
+    class RepliesSerializer < ActiveModel::Serializer
+      attributes :type, :total_items
```

Comment:
The ActivityStreams spec is actually not clear about that, but in all examples, the options are just `Object`s identified by their `name`, and voting is simply done by sending an `Object` with the same `name` as the option `inReplyTo` the `Question`.
I think we should do the same.
Also, we should work with Pleroma to make sure we are interoperable, as they are currently investigating about polls as well.

Reply:
Apparently Misskey already implemented federated polls, so that's what we should match. The voting is implemented like you said.

Reply:
Should be noted Misskey serializes vote results as `_misskey_votes` property. I think using `replies` with `totalItems` is more in the spirit of the protocol so I will go with that :stuck_out_tongue: